com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/VALUE.cs

```diff
@@ -0,0 +1,7 @@
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class VALUE
+    {
+        
+    }
+}
```

Comment:
Is this intended for use?

Reply:
It was needed for completing the idea of having default values that aren't REFs.

Reply:
Removed this file. We will address this need when it comes up.